fireball/app/_components/filterComponents/FilterInputs.jsx

```diff
@@ -1,40 +1,23 @@
-import meteoriteData from "public/meteoriteData.json";
 export default function FilterInputs(props) {
   const {
-    setFilteredDataSet,
     nameSearch,
     recSearch,
     setNameSearch,
     setRecSearch,
   } = props;
 
-  // Function to handle text input for Name field
+  // On change handler for text input for Name field
   function handleNameInput(e) {
     const newName = e.target.value;
     setNameSearch(newName);
-    filterDataSet(newName, recSearch);
   }
 
-  // Function to handle text input for RecClass field
+  // On change handlerfor text input for RecClass field
   function handleRecInput(e) {
     const newRec = e.target.value;
     setRecSearch(newRec);
-    filterDataSet(nameSearch, newRec);
   }
 
-  // Triggered from filter inputs to manipulate data variable //
-  function filterDataSet(nameInput, recInput) {
-    const filteredData = [...meteoriteData]
-      .filter((meteorObj) => {
-        return meteorObj.name.toLowerCase().includes(nameInput.toLowerCase());
```

Comment:
I noticed that currently, the filter does not work when a filter keyword is capitalized. It's likely due to the fact that you're normalizing the search terms to lowercase before performing the filtering. 

I tested and this code should fix it. 

return (   meteorObj.name.toLowerCase().includes(nameInput.toLowerCase()) ||
          meteorObj.name.includes(nameInput)
        );

Let me know if you need clarifications.

Reply:
I took that out when I was looking at something else, and must of forgot to put it back in.  I'll fix in a moment

Good catch!